.travis.yml

```diff
@@ -42,6 +43,7 @@ matrix:
           - uuid-dev
           - doxygen
           - openssl
+          - ninja-build
```

Comment:
I really like ninja and I am happy that it is added, but am I correct that we don't actually use it on Travis? I don't see the  -g/--useninja flag specified anywhere.

Reply:
Correct.

Reply:
We used it on Windows which helped, but there is a dependency issue I did not aim to solve here. Locally I addressed that issue but didn't see a significant speed up over cmake build ( which more apply generates external project builds in parallel as Ninja does ) -- so I will remove Ninja from the travis here. I tried many incremental changes so some were left in here like this. I will remove this line as you are correct it does not serve a purpose.

Reply:
That's perfect.